.github/workflows/doc-issues.yaml

```diff
@@ -0,0 +1,46 @@
+name:                Create doc issues on PR merge
+
+# Action that creates issues in cockroachdb/docs when a PR is merged from
+# cockroachdb/cockroach in the master or release-* branch and has a
+# release note in the PR body.
+
+on:
+  pull_request:
+    branches:
+      - master
+      - 'release-*'
+    types:           [closed]
```

Comment:
passing comment: is `[closed]` also when someone closes a PR but does not merge it?

Reply:
The action will trigger in that scenario, yes, but the job has a check to see if the PR was merged on L17.

```
    if:              ${{ github.event.pull_request.merged == true }}
```

GitHub Actions doesn't have a PR merge event, only a close event.

Reply:
Nice!